pkg/drain/podDeleteStrategy.go

```diff
@@ -16,7 +13,8 @@ type podDeletionStrategy struct {
 }
 
 func (pds *podDeletionStrategy) Execute(node *corev1.Node) (*DrainStrategyResult, error) {
-	podsToDelete, err := pds.getPodList(node)
+	filters := []pod.PodPredicate{isOnNode(node)}
```

Comment:
Any reason why no appending of the podDeletionStrategy filters here? ie.
```suggestion
	filters := append([]pod.PodPredicate{isOnNode(node)}, pds.filters...)
```
That seems to be inadvertently being dropped through this change. I notice it is being done for the `stuckTerminatingStrategy.go` file, but not here.